libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx

```diff
@@ -40,7 +40,39 @@ function HomeTabFeatured() {
               autoPlaySpeed={15000}
               dotListClass="position-relative mt-2"
             >
-              <div className="mx-1 px-1 d-flex">
+              <div className="mr-1 pr-1 d-flex">
+                <a href="https://cryptpad.fr/form/#/2/form/view/pV-DdryeJoYUWvW+gXsFaMNynEY7t5mUsgeD1urgwSE/" target="__blank">
+                  <img src={'/assets/img/soliditySurvey2023.webp'} style={{flex: '1', height: '170px', maxWidth: '170px'}} alt=""></img>
+                </a>
+                <div className="h6 w-50 p-2 pl-4  align-self-center" style={{flex: '1'}}>
+                  <h5>
+                  SOLIDITY DEVELOPER SURVEY 2023
+                  </h5>
+                  <p className='pt-2 mb-1'>
+                    Please take a few minutes of your time to complete the survey.
+                  </p>
+                  <p className="mb-3">
+                    Thank you for your support! Read the full announcement
+                    <a
+                      className="remixui_home_text mx-1"
+                      onClick={() => _paq.push(['trackEvent', 'hometab', 'featuredSection', 'soliditySurvey23'])}
+                      target="__blank"
+                      href="https://soliditylang.org/blog/2023/12/08/solidity-developer-survey-2023-announcement/"
+                    >
+                      here.
```

Comment:
`here` text is small comparing to complete line

Reply:
it's a link